en/02_Developer_Guides/08_Performance/06_ORM.md

```diff
@@ -21,7 +21,7 @@ SilverStripe\Forms\TreeDropdownField:
 ```
 
 > [!TIP]
-> A very common use of `TreeDropdownField` is the "Insert Link" feature in the TinyMCE WYSIWYG. Setting this configuration to use another filter and adding an index on `Title` and `MenuTitle` for [`SiteTree`](api:SilverStripe\CMS\Model\SiteTree) can significantly improve performance.
+> A very common use of `TreeDropdownField` is the "Insert Link" feature used by supported HTML editors. Setting this configuration to use another filter and adding an index on `Title` and `MenuTitle` for [`SiteTree`](api:SilverStripe\CMS\Model\SiteTree) can significantly improve performance.
```

Comment:
I'm making an assumption here that a replacement for TinyMCE will have similar plugins which use the same forms and modals our TinyMCE ones do. I think that's a safe assumption.